fix: group enable/disable endpoints need url param

caproven opened the pull request:
Quick fix for receiver group enable/disable endpoints, which don't seem to currently work. I originally tried using query params (ex. `api/v1/groups/enable?groupID=myid`) to no avail. It sems `chi.urlParam()` only fetches params provided in routing such as `r.Route("/{param}", handler)`.

With change, endpoints are hit with
- PUT `api/v1/groups/GROUP_ID/enable`
- PUT `api/v1/groups/GROUP_ID/disable`

xbcsmith commented:
Need to update the doc to reflect this feature.

gbuskey commented:
im pretty sure this is happening because our import are messed up. can you try reverting your change and updating our chi import paths to use v5?

caproven commented:
> im pretty sure this is happening because our import are messed up. can you try reverting your change and updating our chi import paths to use v5?

I tried this and it didn't work - unless I changed the handler this was the only way I could get things working.

> Need to update the doc to reflect this feature.

Wasn't sure where exactly to doc this. Added a piece in the glossary, please see if that suffices for now.

xbcsmith commented:
> > im pretty sure this is happening because our import are messed up. can you try reverting your change and updating our chi import paths to use v5?
> 
> I tried this and it didn't work - unless I changed the handler this was the only way I could get things working.
> 
> > Need to update the doc to reflect this feature.
> 
> Wasn't sure where exactly to doc this. Added a piece in the glossary, please see if that suffices for now.

Doc goes in reference. Probably should start an api.md or something along those lines

xbcsmith reviewed (changes requested):
We should change this to a PATCH on /groups/{ID} to enable/disable

ninjamast3r commented on pkg/api/api.go:
```diff
@@ -90,10 +90,10 @@ func Initialize(db *storage.Database, msgProducer message.TopicProducer, cfg *co
-				r.Put("/disable", s.Rest.SetGroupEnabled(false))
 				r.Route("/{groupID}", func(r chi.Router) {
 					r.Get("/", s.Rest.GetGroupByID())
+					r.Put("/enable", s.Rest.SetGroupEnabled(true))
```
The team talked about this on our Friday call. We think the better solution is is to do a PATCH request to the /{groupID} with the enabled field changed. After thinking about it, that seemed like the more REST-y solution.

caproven replied:
Switched to PATCH `/groups/{groupID}`, please take a look at implementation

ninjamast3r reviewed (commented):
Looks ok to me. Rebase, and I'll approve it.
